python/py-buildbot-macports-custom-views/Portfile

```diff
@@ -4,31 +4,30 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-buildbot-macports-custom-views
-version             0.1.1
+version             0.1.2
 revision            0
-
-maintainers         {rajdeep @rajdeepbharati} openmaintainer
-
-description         MacPorts Custom Views Plugin for Buildbot
-long_description    ${description}
+checksums           rmd160  fbc1980b7fcea013fa0a4ed50b7eb88426223579 \
+                    sha256  0ac7f8de66102dd3f938382aec3cccd8a3c764b3773d82914c84f181b0123891 \
+                    size    447411
 
 platforms           darwin
-supported_archs     noarch
+maintainers         {rajdeep @rajdeepbharati} openmaintainer
 license             MIT
+supported_archs     noarch
 
-homepage            https://github.com/macports/buildbot-macports-custom-views
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
-distname            ${python.rootname}-${version}
+description         MacPorts custom views plugin for Buildbot web UI
+long_description    {*}${description}
 
-checksums           sha256  4fcc176f033f5b838dd7376628ef2054053f3b8f51560940e510e75edea23a5c \
-                    rmd160  cd224aaa98114324b046094f6d7b2aac8b31389a \
-                    size    447418
+homepage            https://github.com/macports/buildbot-macports-custom-views
```

Comment:
Please note that most of these newer ports have likely been generated with upt. If you feel that defaults from upt could be improved, please suggest changes. @reneeotten

Reply:
I've never looked at upt. I can't look at everything. :)